src/app/shapes/Shape.ts

```diff
@@ -25,14 +25,15 @@ export abstract class Shape {
     }
     
     public remove(): void {
-        // TODO
+        this.instance.destroy();
     }
     
-    public move(): void {
-        // TODO
+    public move(x: number, y: number): void {
+        this.instance.x(x);
+        this.instance.x(y);
     }
```

Comment:
Probably need to test this because behavior might be different for various shapes.